cfme/utils/credentials.py

```diff
@@ -0,0 +1,10 @@
+from dynaconf import settings
```

Comment:
This would require us just to update all our imports 
FROM  `from cfme.utils.conf import credentials`  TO `from cfme.utils.credentials import credentials`